src/game/lara/cheat.c

```diff
@@ -374,7 +374,11 @@ bool Lara_Cheat_Teleport(int32_t x, int32_t y, int32_t z)
         g_Lara.gun_status = LGS_ARMLESS;
     }
 
-    Lara_GetOffVehicle();
+    if (g_Lara.skidoo != NO_ITEM) {
+        Lara_GetOffVehicle();
+        g_LaraItem->rot.x = 0;
+        g_LaraItem->rot.z = 0;
```

Comment:
I believe the new assignments would best belong inside `Lara_GetOffVehicle`. Additionally, since the check is already done within the function, it can be safely omitted.